Client/Frontend/Browser/OpenSearch.swift

```diff
@@ -282,4 +282,4 @@ class OpenSearchParser {
 
         return OpenSearchEngine(engineID: engineID, shortName: shortName, image: uiImage, searchTemplate: searchTemplate, suggestTemplate: suggestTemplate, isCustomEngine: false)
     }
-}
\ No newline at end of file
+}
```

Comment:
I know sometimes xcode does this. But to keep commits clean don't checkin these files when only one line has changed.